e2e/gui/src/test/java/com/epam/pipeline/autotests/RestrictionsOnInstancePriceTypeTest.java

```diff
@@ -188,123 +693,24 @@ public void validationOfInstanceTypesRestrictionsExistingObjects() {
                 .exitFromConfigurationWithoutSaved();
     }
 
-    @Test(priority = 2, dependsOnMethods = {"preparationForValidationOfInstanceTypesRestrictions"})
-    @TestCase({"EPMCMBIBPC-2639"})
-    public void validationOfInstanceTypesRestrictionsCreatingObjects() {
-        loginAs(admin);
-        setMaskForUser(user.login, instanceTypesMask, format("%s.*", instanceFamilyName));
-        logout();
-        loginAs(user);
+    private void validationOfPriceTypesRestrictions(String pipelinePriceTypes, String configurationPriceTypes, String toolPriceTypes) {
         library()
                 .cd(folder)
-                .createPipeline(secondPipeline)
-                .clickOnDraftVersion(secondPipeline)
+                .clickOnDraftVersion(pipeline)
                 .configurationTab()
-                .editConfiguration("default", profile ->
+                .editConfiguration(configurationName,profile->
```

Comment:
Could you please add space after `,` and before `->`?

Reply:
done